corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html

```diff
@@ -204,7 +204,7 @@
     </tr>
 </table>
 <div class="col-md-6">
-    <h2>{% trans "Active Users Trend" %}</h2>
+    <h2>{% trans "Active Users Trend" %} <small><a href="{% url 'domain_internal_settings' domain %}">{% trans "Set Performance Threshold" %}</a></small></h2>
```

Comment:
it looks like this is a superuser only view right now, so this link should also probably only show up for superusers. when we "release" the report we should make both public